rdflib/namespace.py

```diff
@@ -95,11 +95,6 @@ def __new__(cls, value):
             rt = unicode.__new__(cls, value, 'utf-8')
         return rt
 
-
-    @property
-    def title(self):
-        return URIRef(self + 'title')
-
```

Comment:
I agree with should restore it, but this is a problem for ask methods on Unicode objects, lower, islower, index etc